### PR TITLE
Igal/mgmt 1382

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -141,6 +141,7 @@ var clusterFileNames = []string{
 	"metadata.json",
 	"kubeadmin-password",
 	"kubeconfig-noingress",
+	"install-config.yaml",
```
Comment (romfreiman) [inline internal/bminventory/inventory.go:144]: ha? That's it? :)

Comment (romfreiman) [reply internal/bminventory/inventory.go:144]: Who uploads the file to s3?

Comment (tsorya) [reply internal/bminventory/inventory.go:144]: from Osher's repo. All the changes were there

Comment (filanov) [reply internal/bminventory/inventory.go:144]: What does this file contain?

Comment (eranco74) [reply internal/bminventory/inventory.go:144]: The cluster install config (what we pass to kubeconfig-ignition-generator...)

**Pull request merged**